### DISCUSSION TITLE
my academic github.io page does not show up in google search

### DISCUSSION THREAD

**Discussion by guiwang1201** [community/community#65479] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi,

I have created a personal website using Jekyll. (Github repo: **https://github.com/academicpages/academicpages.github.io**). My website is: **https://guiwang1201.github.io/**.

It has passed more than 7 months and still have not showed up in google search. Could you please help?

Thank you very much in advance.

**Comment by contsili** [comment]
Hi @guiwang1201,

It takes time for Google Search engine's web crawlers to index a website in the search engine. As far as I know, the best practice to speed up this process is to "advertise" your website in social media etc. More information can be found in an older discussion: https://github.com/orgs/community/discussions/44421.

Hope that helps!

**Comment by seyLu** [comment]
#### You can manually request google to crawl your website by:
- going to [Google Search Console](https://search.google.com/search-console/about)  
- register the domain as your property (in this case guiwang1201.github.io)
- and verify that you own the domain

#### It might take some more time (can be days, can be weeks, can be months, just be patient) before your site gets shown in google search.

#### You can also read more on the references below, or the older github discussion as pointed by @contsii

references:
[Setting up Google search console for Jekyl github pages](https://www.aleksandrhovhannisyan.com/blog/getting-started-with-jekyll-and-github-pages/#setting-up-google-search-console)
[Jekyl github page not found in google](https://stackoverflow.com/questions/49212888/github-page-not-found-in-google)
[github pages not found in search results](https://stackoverflow.com/questions/49073043/get-github-pages-site-found-in-google-search-results)

**Reply by guiwang1201** [reply]
Thanks @seyLu.

Actually I have already verified that I own the domain(guiwang1201.github.io) through HTML about two months ago. 

However, i still could not search my website on Google.![Screenshot_2023-08-31-14-50-24-659_com.microsoft.emmx-edit.jpg](https://github.com/community/community/assets/112051244/f01c5ccd-0efb-4fd1-9d99-dee5a37cc201)

**Reply by seyLu** [reply, marked as answer]
### Hmmm, if that is the case, you might want to add backlinks to your site
#### Take a site that is already indexed by google and add a link that points to your site (it can be of any):
- create a youtube vid/twitch stream and add your site link to the description
- write blogs/articles in dev.to, medium, etc. and make sure to add your site link
- in your github, link to your site
- your socials (twitter/fb/reddit/etc.), add a link that points to your site
- just any site really that is already indexed by google -- do this as many times

**Reply by guiwang1201** [reply]
Thanks, I will have a try*

**Reply by guiwang1201** [reply]
It works! Thanks!

**Comment by laurensvdwiel** [comment]
I want to share my experience regarding indexing issues after adding my page to Google Search Console. 

I encountered problems caused by internal redirects due to the use of `redirect_from:` in the headers of my markdown (.md) pages. These redirects were included in the `sitemap.xml`, which Google Search Engine crawlers indexed. However, the internal redirect pages in Jekyll/AcademicPages were created as temporary pages with a `noindex` HTML meta tag, leading to further indexing issues.

After removing all instances of `redirect_from:` across my AcademicPages in the `_pages` directory, my page is now appearing again in the top search results.
